homeassistant/components/media_player/samsungtv.py

```diff
@@ -0,0 +1,175 @@
+"""
+homeassistant.components.media_player.denon
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provides an interface to Samsung TV with a Laninterface.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.samsungtv/
+"""
+import logging
+import socket
+from samsungctl import Remote
```

Comment:
After you've fixed this, run `script/gen_requirements_all.py` to add your requirement to `requirements_all.txt`.